src/ui/services/products/modalWindow.service.ts

```diff
@@ -0,0 +1,34 @@
+import { ModalWindowPage } from "../../pages/products/modalWindow.page.js";
+import { textNotification } from "../../../data/modalWindowTextNotification.js";
+import { ProductsPage } from "../../pages/products/products.page.js";
+import { IProduct } from "../../../data/types/product.types.js";
+
+export class ModalWindowService {
+    constructor (
+        private modalWindowPage = new ModalWindowPage(),
+        private productsPage = new ProductsPage(),
+    ) {}
+
+    async checkNotificationCreateProduct() {
+        expect(await this.modalWindowPage.getNotificationText()).toEqual(textNotification.createProduct)
+        await this.modalWindowPage.closeNotificationWindow()
+    }
```

Comment:
На будущее нужно будет предусмотреть случай, когда подряд будет несколько тостеров
```suggestion
    async checkNotification(text: string) {
        expect(await this.modalWindowPage.getNotificationText()).toEqual(text)
        await this.modalWindowPage.closeNotificationWindow()
    }
```

Reply:
в след. ДЗ сделаю